src/modules/feature-modules/admin/pages/service-users/service-users-find.component.ts

```diff
@@ -14,13 +16,35 @@ export class PageServiceUsersFindComponent extends CoreComponent implements OnIn
     { type: 'button', label: 'New user', url: '/admin/service-users/new' }
   ];
 
-  constructor() {
+  form = new FormGroup({
+    search: new FormControl()
+  });
 
-    super();
-    this.setPageTitle('Service users');
+  serviceUsers?: searchUserEndpointDTO[];
+
+  searching = false;
+
+  constructor(
+    private serviceUsersService: ServiceUsersService
+  ) {
 
+    super();
+    this.setPageTitle('Service users search');
+    this.serviceUsers = [];
   }
 
   ngOnInit(): void { }
 
+  onSubmit(): void {
+    this.searching = true;
+    const email = this.form.get('search')!.value;
+    this.form.setValue({search: ''});
```

Comment:
Good point. The reasoning is because this is an exact match field, so if you want to make a new query, it saves the user from having to select all and delete. 
But it's up for consideration, this was a personal decision